### ISSUE TITLE
Isn't automatically mounting ux0

### ISSUE THREAD

**Issue by MessiahMozgus** [SKGleba/yamt-vita#2]
3.65 Enso on a Vita 2000 here. Disabled gamesd.skprx, rebooted, then installed the vpk and ran it. It closes in a flash but it does add the plugin file to boot_config and puts the file in the right place. Rebooted. The SD card is never mounted on boot. I have to mount it from VitaShell and then refresh livearea to get the content to show. After every reboot, I must then remount it from within VitaShell.

**Comment by AbFarid** [comment]
Same exact situation.
Another problem is that the app closes so fast, you can't read the output, even though 2 different things can happen when you launch the app.
The lines printed on screen are either:
```
Found yamt.skprx in ur0:tai/ !
removing... ok!
restoring old boot_config.txt... ok!
done!
```
or:
```
Could not find yamt.skprx in ur0:tai/ !
copying... ok!
adding entry to ur0:tai/boot_config.txt... ok!
done!
```

**Comment by MessiahMozgus** [comment]
Thats funny. I was trying to record it with my phone camera and it couldnt focus on the text in time to get even a frame of video that was readable. But I figured thats what it was doing.

**Comment by AbFarid** [comment]
I was trying slow-mo recording, but couldn't focus, too, so I used burst-photo instead and it worked.

**Comment by MessiahMozgus** [comment]
I just downgraded to 3.60 since this motivated me. Formatted internal storage. Deleted tai folders on sd and internal before installing a fresh henkaku and then enso. The issue remains. Hope you can fix this.

**Comment by alpakeno** [comment]
Do restore all settings from recovery to restore the boot_config.txt file, it may fix your issue.

**Comment by MessiahMozgus** [comment]
Follow up. I did a reset of all settings from the green settings menu. I didnt find one in the recovery menu so I figured this was the same thing. Also just for the hell of it, I formatted everything again, and plugged in a 4GB sony mem card to use instead of the 1GB internal on Vita 2000. So formatted both, setup the system again, henkaku 3.60, enso, now your plugin, and so far it is working! Mounts my SD card on boot with mem card as uma0. Nice. I can even still mount the 1GB internal as imc0.

So either a system settings reset did the trick, or switching to a 4GB sony mem card for the initial setup did.
